tutorials/clustering.md

```diff
@@ -63,37 +63,54 @@ using our informational views:
 SELECT * FROM timescaledb_information.data_node;
 ```
 
-Now that we've created a database and connected a couple data nodes, let's go ahead
-and create a distributed hypertable. With the following commands, we'll create
-a hypertable and then distribute it:
+Now that we've created a database and connected a couple data nodes,
+let's go ahead and create a distributed hypertable. With the following
+commands, we'll create a distributed hypertable for collecting
```

Comment:
When working with an editor, I was constantly reminded that I should use "you" instead of "we" and "us". :) It would be a lot to change if you did it in this chapter though, so feel free to ignore it.

Reply:
Leaving as is for now. But, fwiw, in my experience "we" is more common, at least when you want to give the impression that you (as an author) are going through the steps together with the reader. At least in academic papers this seems to be the norm (have never encountered "you" in that context). Maybe our American friends can chime in here? @WireBaron

Reply:
Yes, it is what I was taught when writing research papers as well, but my editor claimed my writing style was too academic and only what he saw in academic circles.

Reply:
I guess that fact that we're a bunch of academics should be reflected somewhere :)